bucket/liteide.json

```diff
@@ -18,6 +18,6 @@
         "regex": "tag/x([\\d\\.]+)"
     },
     "autoupdate": {
-        "url": "https://downloads.sourceforge.net/project/liteide/x$version/liteidex$version.windows-qt5.9.5.zip"
+        "url": "https://github.com/visualfc/liteide/releases/download/x$version/liteidex$version.windows-qt5.9.5.zip"
```

Comment:
Would not make sense to match qt version?

How long is qt lifecycle in this application?

Reply:
You are right. It would make more sense to match Qt version in `checkver`.
Their last Qt update was [about a year ago](https://sourceforge.net/projects/liteide/files/x34/).

Reply:
Stop resolving conversations immidiatelly after you comment ...

https://github.com/lukesampson/scoop-extras/pull/3056#discussion_r338548603

Reply:
Okay. Didn't know that. Thanks for noticing!